src/util/file.cpp

```diff
@@ -90,33 +90,41 @@ StringFile::StringFile(const char *filename_, std::string &&content_)
 #ifdef __EMSCRIPTEN__
 #include <emscripten/emscripten.h>
 
+// clang-format off
+EM_ASYNC_JS(int, getBase, (int *length, const char *filename), {
```

Comment:
Why does this return `int`? Why not make it return uint8_t* instead?

Reply:
Done